### ISSUE TITLE
VAMC real-time banners: Environmental Configs for Data Sync from Drupal

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#19918]
## Description
Banners are imported to vets-api every 10 minutes via Sidekiq import scripts. These scripts run on each environment: dev, stage, and prod. For each environment, the data should be pulled from a respective Drupal environment (or tugboat equivalent).

In order to do this we need to use settings overrides and parameter store secrets much like va_forms does as described in the Platform [documentation](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-VetsAPIsettingsandsecrets).

Ideally both the banners and va_forms modules would pull from the same Drupal instance, but we don't actually know where forms are pulled from in lower environments. To know this, we need to request access to the parameter store, or request them from Platform, or both.

Environment map:
Local: ??
Dev: ??
Stage: staging.cms.va.gov
Prod: prod.cms.va.gov

Parameter Store variables to set:
/dsva-vagov/vets-api/dev/banners_drupal_url
/dsva-vagov/vets-api/staging/banners_drupal_url
/dsva-vagov/vets-api/sandbox/banners_drupal_url
/dsva-vagov/vets-api/prod/banners_drupal_url
/cms/prod/drupal_api_users/banners_api/password
/dsva-vagov/vets-api/dev/banners_drupal_password
/cms/prod/drupal_api_users/banners_api/username

## Acceptance criteria
This isn't really testable on its own, since we are just updating a manifest here. Full testing of this will happen in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19452

- [x] Manifest is updated with settings overrides
- [x] Manifest is updated with config overrides
- [x] Parameter Store values have been created

**Comment by dsasser** [comment]
I requested access to the parameter store [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97546).

**Comment by dsasser** [comment]
PR submitted: https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3276

**Comment by dsasser** [comment]
> Note that this work will require two PRs per platform requirements:
> 
> 1. Settings for lower environments. [Submitted](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3276), ~~pending review.~~ merged 11/22/24
> 2. Settings for Prod only. Local changes staged, awaiting first PR approval.

There is one additional requirement here, in that the Parameter Store values are created. We have requested access, but as launch approaches, if we still lack access, we'll need to ask Platform to enter the values for us.

Keep this issue alive until the Parameter Store values have been stored, less we lose track of that activity.

**Comment by dsasser** [comment]
The first PR failed deployment, and needed to be [reverted](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3282). This reavealed that, before we can merge config changes, we need to get the AWS parameter store values in first, or we get errors during deploy such as:

```
ParameterNotFound: /cms/prod/drupal_api_users/banners_api/username could not be found.
```

**Comment by jilladams** [comment]
cc @Agile6MSkinner @SnowboardTechie  re: the 3 comments above this, where: 
* This ticket is launch blocking
* We need an engineer to get AWS access in order to do it ourselves
* Without that: we will need Platform help to make the env config changes. 

I am not totally clear on whether this ticket blocks the last FE ticket to hook up the vets-website app to the API. 
If not: we're probably ok, and Daniel can come back to this after his PTO. 
If so: we might need Bryan to work on AWS access, and/or collab with Platform to get this moved, to unblock the last FE work.

**Comment by SnowboardTechie** [comment]
New PR to merge in the env configs https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3287

**Comment by SnowboardTechie** [comment]
One sandbox URL was missed during the slack conversations yesterday so I put in a platform support request [here](https://dsva.slack.com/archives/CBU0KDSB1/p1732631548907429) to add this before https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3287 can be opened for review and merged

**Comment by SnowboardTechie** [comment]
I requested access to the AWS param store https://github.com/department-of-veterans-affairs/va.gov-team/issues/97859

**Comment by dsasser** [comment]
## Mid-sprint Update 12/2/24

Wiating for platform approval on the [PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3287).

**Comment by SnowboardTechie** [comment]
This is now merged and can be confirmed once #19451 is as well

**Comment by SnowboardTechie** [comment]
[Support request opened](https://dsva.slack.com/archives/CBU0KDSB1/p1733764791171769) for platform PR review

**Comment by SnowboardTechie** [comment]
Production ENV keys for banners is now merged. Given the time of day, I'm assuming this will not make it live until tomorrow's deploy.

**Comment by SnowboardTechie** [comment]
Prod ENV keys are now live and functioning
